README.rst

```diff
@@ -51,25 +51,20 @@ Using the ``--path`` argument, you can specify a different site path than ``.``:
 
     $ bin/serve --path /path/to/cactus/site
 
+Crate Cactus GUI/CLI
+====================
 
-Running GUI and building standalone executable
-==============================================
-
-Requires::
-
-    Python 3.4, Tcl/Tk + Python bindings
-
-On OSX+MacPorts install::
-
-    $ sudo port install tk +quartz py34-tkinter jpeg openjpeg webp
-
-To run the GUI::
+Starting the GUI requires Python with Tcl/Tk support (tkinter).
```

Comment:
This is a prerequisite? Maybe refer to an article on how to install? Or is it just pip?